server/main.go

```diff
@@ -102,6 +105,16 @@ func main() {
 		glog.Fatalln("Cannot config global logger, use default one: ", zap.Error(err))
 	}
 
+	defer func() {
+		if r := recover(); r != nil {
+			log.S().Errorf("crashlog: %v", r)
```

Comment:
move into `recovery.CrashLog()`

Reply:
`recover()` can not  act if move into `recovery.CrashLog()`

Reply:
```
if r := recover(); r != nil {
	recovery.CrashLog(r, cfg.Log,)
}
```

Reply:
ok, it's clearly